src/main/resources/org/boozallen/plugins/jte/hooks/Invoke.groovy

```diff
@@ -2,8 +2,19 @@ import org.boozallen.plugins.jte.hooks.Hooks
 import org.boozallen.plugins.jte.binding.TemplateBinding
 import org.boozallen.plugins.jte.hooks.AnnotatedMethod
 import java.lang.annotation.Annotation
-    
+
 void call(Class<? extends Annotation> a, TemplateBinding b, Map context){
-    List<AnnotatedMethod> discovered = Hooks.discover(a, b)            
-    discovered.each{ it.invoke(context) }
+    List<AnnotatedMethod> discovered = Hooks.discover(a, b)
+    List exceptions = []
+    discovered.each{
+        try {
+            it.invoke(context)
+        }catch( Exception e){
+            exceptions << e.message
```

Comment:
At this point, you know that the context.status has failed.  so update the context map with the new status.